### DISCUSSION TITLE
Dendro provenance quality/certainty indicator

### DISCUSSION THREAD

**Discussion by johanvonboer** [humlab-sead/sead_change_control#293] [General]
The dendro people requests that the dendro variable "provenance" has a quality flag/indicator - The question is: to what degree are we certain of this provenance?

**Comment by MattiasSealander** [comment]
We are not certain at all. This is information that they would need to provide us, or a way to get it from current data.

**Comment by johanvonboer** [comment]
We will handle it when we get this data.
